client/lib/analytics/index.js

```diff
@@ -356,6 +356,10 @@ const analytics = {
 		setAnonymousUserId: function( anonId ) {
 			window._tkq.push( [ 'identifyAnonUser', anonId ] );
 		},
+
+		optOut: function( isOptingOut ) {
```

Comment:
So `optOut` feels more like a verb to me so calling `optOut( false )` feels a bit odd. I'd recommend changing the name to either changing the name to `setOptOut` or  update the signature to omit the boolean `optOut()` for this action.